README.md

```diff
@@ -51,24 +51,116 @@ If BSG decide to create the ability to do this on live OR I receive a DCMA reque
 
 
 ## SPT-AKI Requirement
-* Stay in Tarkov works requires the [latest AKI Server](https://dev.sp-tarkov.com/SPT-AKI/Server) to run. You can learn about SPT-Aki [here](https://www.sp-tarkov.com/).
-* DO NOT INSTALL THIS ON TO SPT-Aki CLIENT! ONLY INSTALL THE SERVER!
+* Stay in Tarkov requires the [latest AKI Server](https://dev.sp-tarkov.com/SPT-AKI/Server) to facilitate coop connections. You can learn about SPT-Aki [here](https://www.sp-tarkov.com/).
+* You should install the SIT SPT-Aki Server Mod according to the instructions in it's [README.MD](https://github.com/stayintarkov/SIT.Aki-Server-Mod/blob/master/README.md)
+* __**NOTE:**__ The SIT Client Mod should **NOT** be installed onto the SPT-Aki *Client*- it should be installed onto it's own copy of Tarkov. See the Installation>Client section of this readme for further info.
 
 ## [Wiki](https://github.com/stayintarkov/StayInTarkov.Client/blob/master/wiki/Home.md)
 **The Wiki is has been constructed by various contributors. All instructions are also kept within the source in the wiki directory.**
   - ### [Setup Manuals](https://github.com/stayintarkov/StayInTarkov.Client/wiki/Guides-English)
   - ### [FAQs](https://github.com/stayintarkov/StayInTarkov.Client/wiki/FAQs-English)
 
-## Coop
 
-### PREREQUISITE
-You must have the [SPT-Aki mod](https://github.com/stayintarkov/SIT.Aki-Server-Mod) installed in your Server for this module to work. If you do not wish to use the Coop module, you must disable it in the BepInEx config file.
+## Installation
+
+### Overview
+SIT is comprised of 2 major pieces and a launcher
+- A [modified SPT-Aki Server](https://github.com/stayintarkov/SIT.Aki-Server-Mod)
+- The SIT Client Module (this repo!), installed onto an instance of Tarkov
+- [SIT Manager](https://github.com/stayintarkov/SIT.Manager) or [SIT Launcher Classic](https://github.com/stayintarkov/SIT.Launcher.Classic)
+  - You should use the SIT Manager. The classic launcher is only mentioned to clarify for existing classic launcher users.
+
+It's recommended to create the following directory structure to store your SIT Installation. This structure will be referred to for the following sections.
+```
+SIT/
+├── server/      # SPT-Aki Server Mod
+├── game/        # EFT Client
+└── launcher/    # SIT Manager or Classic Launcher
+```
+
+### Server Install
+- Follow the instructions in the SIT SPT-Aki Server Mod repo (linked above) to install and configure the server into the `SIT/server` folder.
```

Comment:
You could just link it here again tbh, not much

Reply:
Agreed.